src/pages/service.tsx

```diff
@@ -161,9 +182,15 @@ const ServicePage = ({ documentTypes }: { documentTypes: DocumentTypes }) => {
     }
   };
 
-  const onVerifyVersion = async () => toggleServiceVerifyDisplayed(true);
+  const onVerifyVersion = async () => showModal('version');
 
   const onValidate = async () => {
+    if (!isContributorFormViewed) {
+      showModal('contributor');
+      setContributorFormViewed(true);
```

Comment:
> So in the end, we are still blocking on the first contribution in the given browser? I am not sure this is a satisfactory experience.

Reply:
Agree, we should not force user to view the contributor modal.

Reply:
I'm pretty sure if we do not do this, the little link below the button saying "Set signature" may not be enough to produce this kind of engagement.

Reply:
I'm pretty sure if we ask for name and email, we will have increased drop rate from uncertain first-time contributors :wink: These are just assumptions against other assumptions… only user testing could tell us that.

What I do know is that in the main use case we have for the contribution tool now, i.e. demonstration to potential partners and then engaged partners, having an additional mandatory step when they first see or try it increases again the felt complexity of an already complex interface, and that once they start working with it on a daily basis it will be very easy to have them set their signature up.

Reply:
I would agree with you @martinratinaud if we were building a system targeting first-time, potential one-shot contributors. This is not the case at the moment 🙂

Reply:
I agree with this point of view and will thus make the changes